js/style/style.js

```diff
@@ -18,12 +18,13 @@ var QueryFeatures = require('../source/query_features');
 var SourceCache = require('../source/source_cache');
 var styleSpec = require('./style_spec');
 var StyleFunction = require('./style_function');
+var getWorkerPool = require('../global_worker_pool');
```

Comment:
Can we replace the global worker pool module with a `var workerPool = new WorkerPool()` in file scope here?